diff --git a/addons/stock/procurement.py b/addons/stock/procurement.py
index 6d1a045..226d422 100644
--- a/addons/stock/procurement.py
+++ b/addons/stock/procurement.py
@@ -460,6 +460,7 @@ class procurement_order(osv.osv):
         product_obj = self.pool.get('product.product')
 
         dom = company_id and [('company_id', '=', company_id)] or []
+        dom.append(('suggest', '=', False))
         orderpoint_ids = orderpoint_obj.search(cr, uid, dom, order="location_id")
         prev_ids = []
         tot_procs = []
